.circleci/config.yml

```diff
@@ -92,13 +92,16 @@ jobs:
 
   check:
     <<: *defaults
+    docker:
+      - image: cimg/python:3.7-node
```

Comment:
This makes the `check` job (and only that one) have Python + the nodejs LTS version so that it can run both python and nodejs stuff.